tests/core/test_wprintf.cpp

```diff
@@ -8,16 +8,19 @@
 #include <stdarg.h>
 #include <wchar.h>
 
+#define MAX_CHARS_SMALL 256
+#define MAX_CHARS_BIG 8096
+
 void PrintWide ( const wchar_t * format, ... )
 {
-  wchar_t buffer[256];
-  memset(buffer, 0, 256);
+  wchar_t buffer[MAX_CHARS_SMALL];
+  memset(buffer, 0, MAX_CHARS_SMALL);
   va_list args;
   va_start ( args, format );
   wprintf(L"format    starts with 0x%x\n", *(int*)format);
   wprintf(L"fmt    continues with 0x%x\n", *(((int*)format) + 1));
   wprintf(L"fmt    continues with 0x%x\n", *(((int*)format) + 2));
-  int r = vswprintf ( buffer, 256, format, args );
+  int r = vswprintf ( buffer, MAX_CHARS_SMALL-1, format, args );
```

Comment:
Looking at the doco the second arg should be the (number of chars in buffer) -1, so I figured I should change that as well.